x/ibc/testing/chain.go

```diff
@@ -349,7 +348,7 @@ func (chain *TestChain) CreateTMClientHeader() ibctmtypes.Header {
 		ChainID:            chain.ChainID,
 		Height:             chain.CurrentHeader.Height,
 		Time:               chain.CurrentHeader.Time,
-		LastBlockID:        MakeBlockID(make([]byte, tmhash.Size), maxInt, make([]byte, tmhash.Size)),
+		LastBlockID:        MakeBlockID(make([]byte, tmhash.Size), math.MaxUint32, make([]byte, tmhash.Size)),
```

Comment:
can we the previous variable? It was there because of some limitations with the math library. cc: @alessio

Reply:
the previous variable does not work in this case as a Uint32 is expected. Im not sure what limiatioin you are talking about as the `math.Max*` variables are just bit shifts like maxInt was

Reply:
I think this field is just a place holder. Pretty sure tests will still pass if it was 0.

Reply:
yup, that is the reason I changed it over. Didn't think it mattered for tests

Reply:
I'm pretty sure it was an OS related issue. I'm fine changing it to another number tho